### ISSUE TITLE
demo is not defined

### ISSUE THREAD

**Issue by HectorTsai** [creativetimofficial/ct-material-dashboard-pro#155]
I have buy material-dashboard-pro and use it for my project, but when test it with sidebar, it show an error "Uncaught ReferenceError: demo is not defined at VM3880 material-dashboard.js:217" and I check the js file. it's code is bellow:

```
// activate collapse right menu when the windows is resized
$(window).resize(function() {
    md.initSidebarsCheck();

    // reset the seq for charts drawing animations
    seq = seq2 = 0;

    setTimeout(function() {
        demo.initDashboardPageCharts();
    }, 500);
});
```

Is anything what shound I do? what is demo object?

**Comment by groovemen** [comment]
Hello @HectorTsai, thank you for using our products, please make sure that you have the demo.js file included in your project, in that file you will find the **initDashboardPageCharts: function()**. We are working on a product update and we'll fix this issue. You can either include the file in your project or you can copy in **material-dashboard.js file** the objects that are called from _demo_ file. Hope this information helps you. Best, Stefan

**Comment by groovemen** [comment]
This issue is fixed in the latest version of the product (V2.0.2).
Best, 
Stefan
